README.md

```diff
@@ -343,6 +343,22 @@ the `profiles` resource).
 The currently supported scopes can be found in the
 [Azure API Authentication middleware](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/utils/middlewares/azure_api_auth.ts#L27).
 
+#### Authorization by client IP
+
+Authorization of API requests can be restricted by client IP.
+This access permission can be configured in the `Service` objects, through the
+`authorizedCidrs` attribute.
+This attribute must contain all the allowed CIDRs and IPs.
+See the documentation of [cidr-matcher](https://www.npmjs.com/package/cidr-matcher)
```

Comment:
It's a `Set` actually, it's part of the `Service` object. You mean specify here the format of a CIDR?

Reply:
I meant to clarify here in the docs if it's a single range of ips or a set and the format of the CIDR (ie. x.x.x.x/24)

Reply:
The attribute has a plural name (`authorizedCidrs`) and the text says "This attribute must contain *all* the allowed CIDRs and IPs" (all plural) - is it not clear that we're talking about multiple CIDRs and IPs ? Also the [CIDR Notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation) is standard and there's plenty examples in the doc I've linked.

Reply:
ok. I said so because I suppose that this input will come from some form / text input developed in a backofice fronted (just to clarify the expected input to the backoffice developers)

Reply:
yes indeed, but that's going to be an issue handled in the backoffice docs/UI - here we're exposing the API that the backoffice will have to call